
syntax = "proto3";

package leapfrogai;

service LLM {
  rpc CountTokens (TokenCountRequest) returns (TokenCountResponse) {}
}

message TokenCountRequest {
  string text = 1;
}

message TokenCountResponse {
  int32 count = 1;
}
